### ISSUE TITLE
Question: How to reset password sample

### ISSUE THREAD

**Issue by PhilippeCorreges** [mean-expert-official/loopback-sdk-builder#641]
#### What type of issue are you creating?
- [ ] Bug
- [ ] Enhancement
- [X] Question

#### What version of this module are you using?
- [X] 2.0.10 (Stable)
- [ ] 2.1.0-rc.n (2.1 Release Candidate n)
- [ ] Other

Write other if any:

#### Please add a description for your issue:
Hello,
I am in the process of developing the password rest functionality client side and I am wondering where I could find the same previously provided by mean expert official on loopback sdk builder. I am pretty sure this sample has this feature in it.
Am I right?

Otherwise I am ok for a nice sample :)

**Comment by gezquinndesign** [comment]
https://loopback.io/doc/en/lb3/User-REST-API.html#reset-password

Hope that helps. As long as you have a User (or User-based) model, the SDK builder will provide the same functionality included in the loopback base model.

**Comment by PhilippeCorreges** [comment]
Hi @gezquinndesign.
Thanks for that. Unfortunately that is not of a big help for me. I am a too much newbee :)

**Comment by gezquinndesign** [comment]
No worries. Have you already used the SDK to implement a login feature? If so, you can use the same feature to reset passwords.

**Comment by PhilippeCorreges** [comment]
Yup, of course. Everything is ok. I am stuck in front of this. I'll tell you as I need to master this.
I nonetheless remember there was a sample somewhere ....

Enjoy your day.

**Comment by gezquinndesign** [comment]
I only ask because they are both in the same API section 
https://github.com/mean-expert-official/loopback-sdk-builder/wiki/5.-Usage-Examples

So in the above example you could set up a method that uses `this.accountApi.resetPasswordRequest()` to setup the request and `this.accountApi.resetAccountPassword()` to actually change it.

Good luck.

**Comment by PhilippeCorreges** [comment]
Hi @gezquinndesign 

I'll tell you once done.
Thanks!

**Comment by PhilippeCorreges** [comment]
Hi @gezquinndesign 
i do not see the above methods in the sample URL you provided. Am I doing wrong?
I nonetheless created a stackoverflow for that: [https://stackoverflow.com/questions/55635072/how-to-implement-reset-password-with-loopback-frameword-and-sdk-builder-client-s](https://stackoverflow.com/questions/55635072/how-to-implement-reset-password-with-loopback-frameword-and-sdk-builder-client-s)

Here is not the good place I think. If someone could help me on stack, please.

Thanks in advance and regards.

**Comment by gezquinndesign** [comment]
I've responded on StackOverflow.  Hopefully it is helpful for you.
